src/NCronJob/NCronJob.csproj

```diff
@@ -37,7 +37,16 @@
 	</PropertyGroup>
 
 	<PropertyGroup Label="Build information">
-		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <!--
+
+      /!\ WARNING.
+
+      Temporary hack to see the CI tests running
+
+      Documentation will be eventually hand rolled before merge.
+
+      -->
+		<GenerateDocumentationFile>false</GenerateDocumentationFile>
```

Comment:
Savagely hacked on purpose. To be reverted before merge

Reply:
Interesting. For what did you need the this?

Reply:
Because I'm (shamelessly) more interested in getting feedback from the CI server test runs on different platforms than documenting the public types.

All the "Missing XML comment for publicly visible type or member xxxx" error messages were making my "Developer Experience" a lot less rich :wink:

Reply:
I am not sure how to properly handle that.

Opting out might lead to missing XML annotations. It is very strict (well at least not if you use DEBUG locally, but that doesn't help in your case)

Reply:
> Savagely hacked on purpose. To be reverted before merge

Once you think the API is ok. I'll write the xml documentation and revert this change. Before the merge.

Reply:
Reverted as documentation has been added.